src/outlineView.ts

```diff
@@ -126,10 +126,9 @@ export class OutlineView {
     this.focusedElms = this.pointToElementsMap.get(cursorPoint)
 
     if (this.focusedElms !== undefined) {
-      this.focusedElms[0].scrollIntoView({
-        block: "center", // scroll until the entry is in the center of outline
-      })
       for (const elm of this.focusedElms) {
+        // @ts-ignore: chrome-only API
```

Comment:
You really should avoid ts-ignore annotations, because those are not selective. They will ignore everything, not only what you intend to ignore. This can (and likely will!) mask bugs down the line.